cc_images/images.yaml

```diff
@@ -64,6 +64,14 @@ CC-Ubuntu24.04:
     release: noble
     variant: base
 
+CC-Ubuntu24.04-hwe:
+  arch:
+    - amd64
+  provenance:
+    distro: ubuntu
+    release: noble
+    variant: base
+
```

Comment:
so this is a complete new image, right? it's separate from CC-Ubuntu24.04 and doesn't cause any increase in those other images if I'm understanding you correctly. Only this `-hwe` image increases in size?

Reply:
yes, this is a completely separate image, it just happens to be bigger because we accidentally include an extra kernel image.

but also, if we publish this _and_ also maintain the non-hwe variant, they shouldn't both be variant:base